bgpd/bgp_zebra.c

```diff
@@ -744,6 +744,9 @@ bool bgp_zebra_nexthop_set(union sockunion *local, union sockunion *remote,
 								? peer->conf_if
 								: peer->ifname,
 							peer->bgp->vrf_id);
+			else if (peer->update_if)
+				ifp = if_lookup_by_name(peer->update_if,
+						peer->bgp->vrf_id);
```

Comment:
please properly align code under peer>update_if

Reply:
I have removed this code since it is part of another commit. please approve.